bin/loadtest.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+. .setup_env
+
+for ((i=0;i<$TEST_WALLETS_TOTAL;i++)); do
+    cd js/tests && WALLET=$i yarn loadtest &
```

Comment:
Bad sign -- complex setup of multiple wallets required for this script, but it is just detail of the implementation.